.github/workflows/ci.yaml

```
@@ -10,9 +10,9 @@ jobs:
       - uses: olafurpg/setup-scala@v10
       - uses: coursier/cache-action@v6
       - name: Lint
-        run: sbt scalafmtCheckAll "rules/scalafix --check"
+        run: sbt scalafmtCheckAll "rules2_12/scalafix --check"
       - name: Test
-        run: sbt coverage +tests/test +rules/coverageReport
+        run: sbt coverage tests/test rules2_12/coverageReport
```

```suggestion
        run: sbt coverage tests/test coverageReport
```
would also work to see coverage across scala versions I guess (like on `master`?) but I wonder how Codecov aggregates several reports?